package.json

```diff
@@ -23,9 +23,15 @@
   ],
   "keywords": [
     "devops",
+    "policy",
```

Comment:
can we get terraform in there too, given Sentinel is a feature of TF.  and hashicorp too I spose :-) Gotta think of the SEO.